contrib/dist/linux/buildrpm.sh

```diff
@@ -471,7 +468,7 @@ cat <<EOF
 ====                FINISHED BUILDING Open MPI RPM                        ====
 ------------------------------------------------------------------------------
 A copy of the tarball is located in: $rpmtopdir/SOURCES/
-The completed rpms are located in:   $rpmtopdir/RPMS/i<something>86/
+The completed rpms are located in:   $rpmtopdir/RPMS/x86_64/
```

Comment:
Can we just say RPMS/<architecture>?  Some of us build on ARM :).

Reply:
+1

Reply:
Fixed.